### DISCUSSION TITLE
Welcome to node-red-contrib-oracledb-mod Discussions!

### DISCUSSION THREAD

**Discussion by vtulluru** [vtulluru/node-red-contrib-oracledb-mod#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akhiljakka** [comment]
Hi Vinay Tulluru,
I am having an issue while using this node (version 0.6.3).
My project is to collect the data from one oracle database and post it in another oracle database, in normal conditions it is working fine but if my internet disconnects, my node red stops and throws the following exception ( requestingNode.log is not a function) as shown in the screenshot. Also in debug node I get ( Oracle query error: DPI-1010: not connected, errorCode:DPI-1010: )
![image](https://user-images.githubusercontent.com/122359005/211534981-87429105-d107-4591-96d0-33dc6717e2cf.png)
My connection settings are:
![image](https://user-images.githubusercontent.com/122359005/211536568-f188c72b-5dc0-4bf4-bd9d-c9657dd069a9.png)



Please check this. I hope even my internet disconnects and reconnects after sometime, the oracle DB should resume without a fault. 


Thank you
Akhil
